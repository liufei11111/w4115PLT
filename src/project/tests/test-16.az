{* structure declaration *}
Int main(Int argc, String argv){
Structure s2 = { a="" };
Structure s3 = { a="a" };
String str;
str = "str";
Structure s4 = { a=str };
Structure s5 = { a=str, b = "b"};
}