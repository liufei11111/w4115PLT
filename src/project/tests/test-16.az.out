stmts:

Structure s1 = {};
Structure s2 = { a="" };
Structure s3 = { a="a" };
string str;
str = "str";
Structure s4 = { a=str };
Structure s5 = { a=str, b = "b"};

fdecls: